src/components/MovieCard/MovieCard.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+const MovieCard = ({title, rating, imageSrc}) => {
+
+    return (
+        <div className='movie-card'>
+            <h3>{title}</h3>
+            <p>{rating}</p>
```

Comment:
It would be good to use a toFixed() or something here so we aren't displaying 5 decimal points! 
It may need to be done as a variable and then have that passed in.